bliss/conf/base_config.yaml

```diff
@@ -165,13 +165,16 @@ predict:
         bands: ${encoder.bands}
         predict_device: ${predict.device}
         predict_crop: ${predict.crop}
-    photo_catalog:
-        _target_: bliss.surveys.sdss.PhotoFullCatalog
-        sdss_dir: ${paths.sdss}
-        run: ${predict.dataset.run}
-        camcol: ${predict.dataset.camcol}
-        fields: ${predict.dataset.fields}
-        bands: ${predict.dataset.bands}
+    # dataset:
```

Comment:
Maybe define `decals_dataset` and `sdss_dataset` at the top level, and then here, within predict, set `dataset: ${decals_dataset}` or `dataset: ${sdss_dataset}`

Reply:
Done.